elements/pfe-footer/src/pfe-footer-lightdom.scss

```diff
@@ -0,0 +1,315 @@
+pfe-footer, pfe-footer::before, pfe-footer::after,
```

Comment:
There's an awful lot of light DOM in this PR and a lot of it is duplication of what the component upgrade is doing.  Maybe this needs a shared-assets sass file where the "source of truth" can live and the lightdom and web component styles can extend from there?  Also, this file needs to be named `--lightdom` to match the naming convention of other components and allow build tools to correctly pick this file up.

Reply:
Yeah, it certainly seems like duplication. A lot of this—at least to my perspective at the time of development—was necessary for the upgrade since there is a layout shift. My goal was to make the lightDOM as simple as possible and style it as close as possible to something acceptable (black w/ white text). The upgrade re-does quite a bit of the look and feel to allow for left-align and centered options, etc.—hence the "duplication". If anyone has a better way of handling this, I'm 100% open to it.

As for naming, yeah, we need to update it to `--lightdom` to match.

Reply:
Are these all no-js styles or are some of them for older browsers?  For Edge and IE11, I'd recommend making use of the `@include browser-query` mixin which will scope the styles for those use-cases.

Reply:
@castastrophe, mostly no-js styles.

Reply:
I wonder if no-js styles could be scoped using `pfe-foo:not(.PFElement)`?  When the component upgrades they all get the PFElement class right?

@kylebuch8 I'm trying to remember...was it the class `.PFElement` or the `pfelement` attribute that was better for Angular?